### PR TITLE
delete: overlapping paragraph and fix summary

### PR DESCRIPTION
**Pull request by milasuperstar**
Signed-off-by: Mila Wu <wuzaixuan@pingcap.com>

### CODE REVIEW

**Comment by milasuperstar** [comment]
PTAL @dcalvin

File: blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md [hunk 1]
```diff
@@ -2,16 +2,13 @@
 slug: /how-a-top-game-company-uses-chaos-engineering-to-improve-testing
 title: 'How a Top Game Company Uses Chaos Engineering to Improve Testing'
 author: Hui Zhang @ Fuxi Lab, NetEase
+summary: NetEase Fuxi AI Lab is China’s first professional game AI research institution. In their search for a Chaos Engineering tool to test their Kubernetes-based AI training platform, they chose Chaos Mesh and have improved their system resiliency ever since.
```
Comment (g1eny0ung) [inline blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md:5]: Refer to https://v2.docusaurus.io/docs/markdown-features#markdown-headers.  Seems you need the `description` field.

File: blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md [hunk 2]
```diff
@@ -2,16 +2,13 @@
...
 image: /img/fuxi-case-banner.jpg
 tags: [Chaos Mesh, Chaos Engineering]
 ---
 
 ![How-a-Top-Game-Company-Uses-Chaos-Engineering-to-Improve-Testing](/img/fuxi-case-banner.jpg)
 
-NetEase Fuxi AI Lab is China’s first professional game AI research institution. In their search for a Chaos Engineering tool to test their Kubernetes-based AI training platform, they chose Chaos Mesh and have improved their system resiliency ever since.
-
-<!--truncate-->
```
Comment (g1eny0ung) [inline blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md:13]: The `<!--truncate-->` and its above content represents a brief description of the blog. These should be preserved.

**Review by g1eny0ung — CHANGES_REQUESTED** [review]

File: blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md (hunk 2 above)
Comment (g1eny0ung) [reply blog/2020-11-26-how-a-top-game-company-uses-chaos-engineering-to-improve-testing.md:13]: https://v2.docusaurus.io/docs/blog#summary-truncation

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Pull request merged**